src/main/java/de/tum/in/www1/artemis/repository/StatisticsRepository.java

```diff
@@ -80,6 +133,16 @@
             """)
     List<Map<String, Object>> getExamParticipations(@Param("startDate") ZonedDateTime startDate, @Param("endDate") ZonedDateTime endDate);
 
+    @Query("""
+            select e.endDate as day, count(se.id) as amount
+            from StudentExam se, Exam e
+            where se.submitted = true and se.exam = e and e.endDate >= :#{#startDate} and e.endDate <= :#{#endDate} and e.course.id = :#{#courseId}
+            group by e.endDate
+            order by e.endDate asc
+            """)
+    List<Map<String, Object>> getExamParticipationsForCourse(@Param("startDate") ZonedDateTime startDate, @Param("endDate") ZonedDateTime endDate,
```

Comment:
please use a custom object here and not a map

Reply:
Done